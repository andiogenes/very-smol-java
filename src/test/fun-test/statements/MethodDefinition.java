class Main {
    int x = 10;

    // Без вызова код в методе не выполняется
    void foo() {
        x = 20;
        x++;
        x++;
        x--;
    }

    // Без вызова код в методе не выполняется
    void bar() {
        x = 30;
        x++;
        x++;
        x--;
    }

    void main() {
        println(x);
    }
}