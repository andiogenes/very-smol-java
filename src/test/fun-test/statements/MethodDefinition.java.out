x : INT = 10 : INT
10
